R/export_list.R

```diff
@@ -76,14 +78,20 @@ function(
         }
         outfiles <- file
     }
-
+    if (is.na(archive_format$compress) & archive_format$file != "") {
```

Comment:
1. Use `&&` (gp)

2. This won't write to new directory:

```r
## if archivea is not created
export_list(mylist, file = paste0("file_", 1:3, ".xlsx"), archive = "archivea")
```

Maybe check and create it first?

Reply:
Also, I suggest adding `normalizePath` to `outfiles` and `archive`. Windows cannot handle "~/" without `normalizePath`

```r
outfiles <- base::normalizePath(outfiles)
archive <- base::normalizePath(archive) ## if it is not ""
```

ropensci/readODS#157